pkg/statestore/leveldb/leveldb.go

```diff
@@ -17,16 +17,16 @@ import (
 	"github.com/syndtr/goleveldb/leveldb/util"
 )
 
-var _ storage.StateStorer = (*store)(nil)
+var _ storage.StateStorer = (*Store)(nil)
 
 // Store uses LevelDB to store values.
-type store struct {
+type Store struct {
```

Comment:
why the export?

Reply:
because `leveldb_test`

Reply:
Maybe to export it in export_test.go? Or just to export there GetSchemaName and keep it unexported in the package.

Reply:
I tried it now again (second attempt). The problem is that the unit test needs to verify the schema name is the correct one, but the statestore constructor returns an interface. The unit test is not a black-box test as a result. This is why it was not a black-box test in localstore either (probably the only places in the codebase where this is necessary). The other idea I have is to unexport this, and just have a helper in the test that loads a new leveldb, and just tries to load the specified key out of the instance, and verify that it matches the latest schema version

Reply:
I didn't think of that :) Added a new commit, let me know if this is what you've meant